tests/dummy/app/templates/button.hbs

```diff
@@ -8,12 +8,13 @@
 {{/paper-toolbar}}
 {{#paper-content class="md-padding demo-buttons"}}
 <div class="doc-content">
-  <p>
+  <div layout="row" flex>
```

Comment:
I'm OK with that as well. Just thought that doing that for the new stuff we write will save a bit of work later.